src/components/Table/style.css

```
@@ -0,0 +1,31 @@
+.table__todo {
+  font-family: 'Trebuchet MS', Arial, Helvetica, sans-serif;
```

```suggestion
*{
  font-family: 'Trebuchet MS', Arial, Helvetica, sans-serif;
  }
```
if you want to use spasafic font you have to use it as global font